.travis.yml

```diff
@@ -65,7 +66,7 @@ before_install:
 - popd
 
 script:
-- ${SPREAD_PATH}/spread -v ${TASK}
+- travis_wait 30 ${SPREAD_PATH}/spread -v ${TASK}
```

Comment:
This shouldn't be necessary? Travis jobs are normally limited to 50 mins, and spread does print the "task running late…" to avoid Travis thinking it got stuck?

Reply:
Yeah, it shouldn't, but for some reason the build got killed. This didn't fix it, though. 😐